packages/keystone/logging/getKeystonePinoOptions.js

```diff
@@ -1,6 +1,7 @@
+const serializers = require('pino-std-serializers')
```

Comment:
This should it in package.json of keystone package

Reply:
Good point, do it right here